src/python/pants/backend/java/dependency_inference/PantsJavaParserLauncher.java

```diff
@@ -0,0 +1,75 @@
+package org.pantsbuild.javaparser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.javaparser.ast.body.TypeDeclaration;
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.expr.Name;
+import com.github.javaparser.ast.ImportDeclaration;
+import com.github.javaparser.ast.PackageDeclaration;
+import com.github.javaparser.StaticJavaParser;
+import java.io.File;
+import java.util.AbstractCollection;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+class Import {
+    public static Import fromImportDeclaration(ImportDeclaration imp) {
+        Import analysisImport = new Import();
+        analysisImport.name = imp.getName().toString();
```

Comment:
Java style nit: These should be set by a constructor which should be called from this function.

Reply:
Done